tex/artikel/Semesterticket.tex

```diff
@@ -1,66 +1,10 @@
-\section{Das Semesterticket -- die neue Freiheit}
```

Comment:
Wohow! Das alles sollte nicht gelöscht werden, oder? ^^

Reply:
natürlich nicht, meinem verständnis nach sollte ich mich mit dem Kultursemesterticket-teil beschäftigen, der schon arg out-of-date war. dass der obere teil fehlt bedeutet vor allem, dass ich dir anscheinend die datei geschickt habe, die nur den neuen unteren teil enthielt

Reply:
Hm, okay.

Reply:
lag ich hier einem missverständniss auf?

Reply:
Nein, ich glaube, das stimmt so. Nur, dass die Datei natürlich nicht ohne den Großteil des Artikels sein sollte ^^ Das kann ich gleich aber einfach beheben.

Reply:
So, ist wiederhergestellt.